pytorch/training/docker/1.8/py3/Dockerfile.cpu

```diff
@@ -163,7 +163,8 @@ RUN HOME_DIR=/root \
  && chmod +x /usr/local/bin/testOSSCompliance \
  && chmod +x ${HOME_DIR}/oss_compliance/generate_oss_compliance.sh \
  && ${HOME_DIR}/oss_compliance/generate_oss_compliance.sh ${HOME_DIR} ${PYTHON} \
- && rm -rf ${HOME_DIR}/oss_compliance*
+ && rm -rf ${HOME_DIR}/oss_compliance* \
+ && /opt/conda/bin/jupyter-kernelspec remove -fy python3
```

Comment:
Please add this as a separate RUN statement after the step that installs sparkmagic.